conf/settings/control/stabilization_att_indi.xml

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE settings SYSTEM "../settings.dtd">
+<settings>
+  <dl_settings>
+    <dl_settings NAME="indi">
```

Comment:
This is a duplicate (old version) that was accidentally included during the merge process. Will remove it (nice catch :) )